tests/GenFu.Tests/TestEntities/Person.cs

```diff
@@ -48,5 +48,8 @@ public string GetMiddleName()
             return _middleName;
         }
 
+        public DateTime? DateOfDeath { get; set; }
+
+        public int? NumberOfCats { get; set; }
```

Comment:
Note that Null Int properties will still not work. I have only implemented DateTime at the moment until I get feedback. If this is accepted for pull then I will code up Int as well.